tests/acceptance/features/webUIFiles/copyPrivateLinks.feature

```diff
@@ -1,4 +1,3 @@
-@skip
 Feature: copy current path as a permanent link
   As a user
   I want to copy the permanent link to the current folder
```

Comment:
change that feature description to describe the current feature, it was changed in https://github.com/owncloud/phoenix/pull/2496